src/components/navigation-header/components/give-feedback-button/give-feedback-button.module.css

```diff
@@ -5,19 +5,19 @@ size is not available on initial server-side render,
 which results in a flash of incorrect layout.
 */
 .iconButtonContainer {
-  display: block;
+	display: block;
 
-  @media (--dev-dot-tablet-up) {
-    display: none;
-  }
+	@media only screen and (min-width: 1401px) {
+		display: none;
+	}
 }
 
 .textButtonContainer {
-  display: none;
+	display: none;
 
-  @media (--dev-dot-tablet-up) {
-    display: block;
-  }
+	@media only screen and (min-width: 1401px) {
```

Comment:
⛏️  Nitpick: I think we may need to tweak this slightly due to the `Registry` link on Terraform pages 😬 We might also need tweaks to the mobile-nav breakpoint?

In addition, maybe we could change the language to `Give feedback`? (Random thought, but I could see the feedback feature as a useful ongoing thing, and seems like it could make sense to keep it in the prominent position for at least a week or two after we're out of "Beta").

![CleanShot 2022-10-04 at 14 12 44](https://user-images.githubusercontent.com/4624598/193894799-94cd12b2-b417-4e5e-85ca-2755f9c4e181.gif)

Another factor we could potentially mitigate might be combining the `Sign in` and `Sign up` buttons (eg maybe we could drop `Sign up`?). However that change seems much farther outside the scope of this PR, so maybe not worth pursuing right away 🤷